content/docs/policies/privacy/extension.md

```diff
@@ -3,7 +3,8 @@ title: Extension Privacy Policy
 toc_title: Extension
 description: This page informs you of our policies regarding the collection, use, and disclosure of personal data when using the Scratch Addons extension and the choices you have associated with that data.
 aliases:
-  - /privacy
+  - /docs/privacy/policies/extension
+  - /privacy/extension
```

Comment:
Can we keep `/privacy` as well? Otherwise it will 404, and linking to one of the policies is better than none?